interfaces/IPositionManager.sol

```diff
@@ -3,7 +3,21 @@
 pragma solidity 0.7.6;
 pragma abicoder v2;
 
+import '@uniswap/v3-periphery/contracts/interfaces/INonfungiblePositionManager.sol';
+
 interface IPositionManager {
+    struct ModuleInfo {
+        bool isActive;
+        bytes data;
+    }
+
+    struct AaveReserve {
+        mapping(uint256 => uint256) positionShares;
+        mapping(uint256 => uint256) tokenIds;
+        uint256[] positionsId;
```

Comment:
delete array, use 0 to aaveIdCounter to cycle mapping of all tokenIds, closed one it will return 0